indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -66,6 +67,12 @@ class AbstractReviewingState(TitledIntEnum):
     mixed = 5
 
 
+class EditTrackMode(int, IndicoEnum):
+    none = 0
```

Comment:
Do we really need this much expressivity? I thought the only two cases we need is **synced** and **not-synced**.